src/main/java/me/jellysquid/mods/lithium/mixin/gen/fast_netherrack_replacement_blobs/NetherrackReplaceBlobsFeatureMixin.java

```diff
@@ -0,0 +1,125 @@
+package me.jellysquid.mods.lithium.mixin.gen.fast_netherrack_replacement_blobs;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Random;
+
+import it.unimi.dsi.fastutil.longs.Long2ReferenceLinkedOpenHashMap;
+import it.unimi.dsi.fastutil.longs.LongArrayList;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.Shadow;
+
+import net.minecraft.block.Block;
+import net.minecraft.block.BlockState;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+import net.minecraft.util.math.Vec3i;
+import net.minecraft.world.ServerWorldAccess;
+import net.minecraft.world.WorldAccess;
+import net.minecraft.world.gen.StructureAccessor;
+import net.minecraft.world.gen.chunk.ChunkGenerator;
+import net.minecraft.world.gen.feature.NetherrackReplaceBlobsFeature;
+import net.minecraft.world.gen.feature.NetherrackReplaceBlobsFeatureConfig;
+
+@Mixin(NetherrackReplaceBlobsFeature.class)
+public abstract class NetherrackReplaceBlobsFeatureMixin {
+	private static final Long2ReferenceLinkedOpenHashMap<LongArrayList> DELTA_CACHE = new Long2ReferenceLinkedOpenHashMap<>();
+
+	static {
+		DELTA_CACHE.defaultReturnValue(null);
+	}
+
+	// Finds start position
+	@Shadow
+	protected static BlockPos method_27107(WorldAccess world, BlockPos.Mutable mutable, Block block) {
+		return null;
+	}
+
+	// Generates the size of this blob
+	@Shadow
+	protected static Vec3i method_27108(Random random, NetherrackReplaceBlobsFeatureConfig config) {
+		return null;
+	}
+
+	/**
+	 * @reason Faster implementation which utilizes a cache of delta positions instead of iterating outwards many times.
+     * This saves many CPU cycles as this feature is called 100 times per chunk in the Basalt Deltas biome.
+	 * @author SuperCoder79
+	 */
+	@Overwrite
+	public boolean generate(ServerWorldAccess world, StructureAccessor structureAccessor, ChunkGenerator chunkGenerator, Random random, BlockPos blockPos, NetherrackReplaceBlobsFeatureConfig config) {
+		// [VanillaCopy] NetherrackReplaceBlobsFeature#generate
+
+	    Block targetBlock = config.target.getBlock();
+
+		// Get start position
+		BlockPos startPos = method_27107(world, blockPos.mutableCopy().method_27158(Direction.Axis.Y, 1, world.getHeight() - 1), targetBlock);
+
+		// Exit early if we couldn't create the start position.
+		if (startPos == null) {
+			return false;
+		} else {
+			boolean didPlace = false;
+
+			// Get the delta positions from size vector
+            LongArrayList deltas = getDeltas(method_27108(random, config));
+			BlockPos.Mutable mutable = new BlockPos.Mutable();
+
+			// Iterate through the delta positions and attempt to place each one.
+			for (long delta : deltas) {
```

Comment:
afaik this will use the standard Java `Iterator<Long>` and box/unbox as it goes. Might be better to use `LongIterator` directly with `.nextLong()`